mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -181,7 +188,8 @@ def get_datetimes(ds, timestr, yearoffset):  # {{{
 
 
 def preprocess_mpas(ds, onlyvars=None, selvals=None, iselvals=None,
-                    timestr='xtime', yearoffset=1849):  # {{{
+                    timestr='xtime', yearoffset=1849,
+                    variable_map=None):  # {{{
```

Comment:
I would leave it because we use mixed case in MPAS.  Also, I don't view this as my library-- probably at this point it is more your library in terms of fractional code!  My goal, of course, is for this to be a group effort.  Sorry about being overly pedantic.